sky/packages/sky/lib/src/widgets/framework.dart

```diff
@@ -486,9 +486,11 @@ final _InactiveElements _inactiveElements = new _InactiveElements();
 typedef void ElementVisitor(Element element);
 
 abstract class BuildContext {
-  InheritedWidget inheritedWidgetOfType(Type targetType);
+  Widget get widget;
   RenderObject findRenderObject();
+  InheritedWidget inheritedWidgetOfType(Type targetType);
   void visitAncestorElements(bool visitor(Element element));
+  void visitChildrenElements(void visitor(Element element));
```

Comment:
This name doesn't quite seem gramatically...  visitElementChildren?  visitChildElements?